src/gallery/slider.js

```diff
@@ -79,12 +80,24 @@ const clientWidth = window.innerWidth,
 			},
 
 			author = mediaInfo.author ? mediaInfo.author : msg( lang, 'gallery-unknown-author' ),
-			link = mediaInfo.filePage
+			link = mediaInfo.filePage,
+			description = getImageDescription(),
+
+			isCaptionExpandable = () => {
```

Comment:
Why is this "caption expansion" business in this PR?

Reply:
Yea I built on top of the caption expansion PR given that they overlap a bit in the code, this is what I meant with the note in this PR description

> Note this PR builds on top of #85, hence the commit list showing all commits. The only commit relevant to focus mode at this point is 26d2743.

I gave a heads up in a daily meeting before opening #86